config/logstash.yml

```diff
@@ -224,6 +224,9 @@
 #xpack.monitoring.elasticsearch.username: logstash_system
 #xpack.monitoring.elasticsearch.password: password
 #xpack.monitoring.elasticsearch.hosts: ["https://es1:9200", "https://es2:9200"]
+# an alternative to hosts + username/password settings is to use cloud_id/cloud_auth
+#xpack.monitoring.elasticsearch.cloud_id: monitoring:xxxxxxxxxx
```

Comment:
Maybe use something like `monitoring_cluster_id:xxxxx` rather than `monitoring:xxxxx` as using `monitoring` here could be ambiguous